include/detail/copied_au.hpp

```diff
@@ -5,10 +5,10 @@
 
 namespace impl_ptr_policy
 {
-    template<typename, typename =std::allocator<void>> struct copied_au;
+    template<typename, typename =std::allocator<void>, typename...> struct copied_au;
```

Comment:
OK, I am not sure I understand what you are trying to do here... You added "typename..." which is not used by the policy. So, now the user can add some parameters that he thinks make difference but those parameters will be silently ignored. Before it'd only take allocator. Anything more would cause an error... which is good IMO. Don't you think?

Reply:
This was to work around getting compiler errors when more template parameters were given for the current policy than every other would accept. Specifically the type aliases for the other policies would fail to compile. Removing this when rebasing.